cli/config.go

```diff
@@ -39,29 +40,29 @@ var configPaths = []string{
 	"keyring.path",
 }
 
-// configFlags is a mapping of config keys to cli flags to bind to.
+// configFlags is a mapping of cli flag names to config keys to bind.
 var configFlags = map[string]string{
-	"log.level":                         "log-level",
-	"log.output":                        "log-output",
-	"log.format":                        "log-format",
-	"log.stacktrace":                    "log-stacktrace",
-	"log.source":                        "log-source",
-	"log.overrides":                     "log-overrides",
-	"log.nocolor":                       "log-no-color",
-	"api.address":                       "url",
-	"datastore.maxtxnretries":           "max-txn-retries",
-	"datastore.store":                   "store",
-	"datastore.badger.valuelogfilesize": "valuelogfilesize",
-	"net.peers":                         "peers",
-	"net.p2paddresses":                  "p2paddr",
-	"net.p2pdisabled":                   "no-p2p",
-	"api.allowed-origins":               "allowed-origins",
-	"api.pubkeypath":                    "pubkeypath",
-	"api.privkeypath":                   "privkeypath",
-	"keyring.namespace":                 "keyring-namespace",
-	"keyring.backend":                   "keyring-backend",
-	"keyring.path":                      "keyring-path",
-	"keyring.disabled":                  "no-keyring",
+	"log-level":         "log.level",
+	"log-output":        "log.output",
+	"log-format":        "log.format",
+	"log-stacktrace":    "log.stacktrace",
+	"log-source":        "log.source",
+	"log-overrides":     "log.overrides",
+	"log-no-color":      "log.nocolor",
```

Comment:
suggestion: Would be nice to have a consistent usage of `no` and `disable` terminology and the `no` prefix to disable a feature.

For example: To disable key ring and p2p users use: `no-keyring` and `no-p2p` respectively, i.e. leading `no` prefix

But for disabling log, it is `log-no-color` rather than `no-log-color`.

Now for config keys we use: `keyring.disabled` `net.p2pdisabled` (the disabled term)

But for log we have `log.nocolor` instead of `log.colordisabled`

IMO would be nice to have at least user facing flags to be predictable / consistent in terminology.